.github/workflows/on-pull-request-opened-and-synchronize.yml

```diff
@@ -12,9 +12,12 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  install_dependencies:
+  install-dependencies:
     name: Install
-    runs-on: macos-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
```

Comment:
So modules for both get cached. Runners that use `ubuntu-latest` and `macos-latest` should [now](https://github.com/CrowdStrike/glide-core/actions/runs/13638042208/job/38121537109#step:3:73) both get a cache hit.